fluent-syntax/tests/ast/mod.rs

```diff
@@ -363,6 +363,32 @@ where
     map.end()
 }
 
+fn serialize_number_literal<'se, S>(raw: &'se str, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let mut map = serializer.serialize_map(Some(3))?;
+    map.serialize_entry("type", "NumberLiteral")?;
+    //map.serialize_entry("raw", raw)?;
```

Comment:
Thanks for implementing a work-around for this. You may want to wait for https://github.com/projectfluent/fluent/pull/242 to land  and uncomment this code.